pkg/operator2/configobservation/interfaces.go

```diff
@@ -11,6 +11,11 @@ import (
 
 var _ configobserver.Listers = Listers{}
 
+const (
+	OAuthServerConfigPrefix    = "oauthServer"
+	OAuthAPIServerConfigPrefix = "oauthServer"
```

Comment:
This should be different, for example `oauthAPIServer`

Reply:
🤦‍♂ copy paste error